src/Akka.Streams.Kafka/reference.conf

```diff
@@ -40,3 +40,15 @@ akka.kafka.default-dispatcher {
   type = "Dispatcher"
   executor = "default-executor"
 }
+
+# Committer flows use this settings to make batch commits
+akka.kafka.committer {
+    # Set max commit batch size
+    max-batch = 1
+    
+    # Set max commit interval
+    max-interval = 1s
+    
+    # Set level of parallelism
+    parallelism = 1
```

Comment:
Is this standard?

Reply:
Mm, no, my bad, did not think that I could check this values out from somewhere... But turned out that they are defined [here](https://doc.akka.io/docs/alpakka-kafka/current/consumer.html)
Updated right now